controllers/list/users/SelectReviewerListHandler.inc.php

```diff
@@ -0,0 +1,140 @@
+ * @ingroup classes_controllers_list
+ *
+ * @brief A class for loading a panel to select a reviewer.
+ */
+import('lib.pkp.controllers.list.SelectListHandler');
+import('classes.core.ServicesContainer');
+
+class SelectReviewerListHandler extends SelectListHandler {
+
+	/** @var int Count of items to retrieve in initial page/request */
+	public $_count = 15;
+
+	/** @var array Query parameters to pass with every GET request */
+	public $_getParams = array(
+		'roleIds' => array(ROLE_ID_REVIEWER),
+	);
+
+	/** @var string Used to generate URLs to API endpoints for this component. */
+	public $_apiPath = 'users/reviewers';
+
+	/** @var array List of user IDs already assigned as a reviewer to this submission */
+	public $_currentlyAssigned = array();
+
+	/**
+	 * @copydoc ListHandler::init()
+	 */
+	public function init($args = array()) {
+		parent::init($args);
+		$this->_count = isset($args['count']) ? (int) $args['count'] : $this->_count;
+		$this->_currentlyAssigned = !empty($args['currentlyAssigned']) ? $args['currentlyAssigned'] : $this->_currentlyAssigned;
+	}
+
+	/**
+	 * @copydoc SelectListHandler::getConfig()
+	 */
+	public function getConfig() {
+
+		$config = parent::getConfig();
+
+		$config['apiPath'] = $this->_apiPath;
+		$config['itemsMax'] = $this->getItemsMax();
+		$config['count'] = $this->_count;
+		$config['currentlyAssigned'] = $this->_currentlyAssigned;
+
+		$config['i18n'] = array_merge($config['i18n'], array(
+			'search' => __('common.search'),
+			'clearSearch' => __('common.clearSearch'),
+			'itemsOfTotal' => __('reviewer.list.itemsOfTotal'),
+			'itemCount' => __('reviewer.list.count'),
+			'loadMore' => __('grid.action.moreItems'),
+			'loading' => __('common.loading'),
+			'filter' => __('common.filter'),
+			'filterAdd' => __('common.filterAdd'),
+			'filterRemove' => __('common.filterRemove'),
+			'filterRating' => __('reviewer.list.filterRating'),
+			'lessThan' => __('common.lessThan'),
+			'moreThan' => __('common.moreThan'),
+			'activeReviews' => __('reviewer.list.activeReviews'),
+			'activeReviewsDescription' => __('reviewer.list.activeReviewsDescription'),
+			'completedReviews' => __('reviewer.list.completedReviews'),
+			'reviewerRating' => __('reviewer.list.reviewerRating'),
+			'daysSinceLastAssignment' => __('reviewer.list.daysSinceLastAssignment'),
+			'daySinceLastAssignment' => __('reviewer.list.daySinceLastAssignment'),
+			'daysSinceLastAssignmentDescription' => __('reviewer.list.daysSinceLastAssignmentDescription'),
+			'averageCompletion' => __('reviewer.list.averageCompletion'),
+			'neverAssigned' => __('reviewer.list.neverAssigned'),
+			'currentlyAssigned' => __('reviewer.list.currentlyAssigned'),
+			'reviewInterests' => __('reviewer.list.reviewInterests'),
+			'gossip' => __('user.gossip'),
+			'biography' => __('reviewer.list.biography'),
+			'listSeparator' => __('common.listSeparator'),
+			'viewMore' => __('review.list.viewMore'),
+			'viewLess' => __('review.list.viewLess'),
```

Comment:
Those should be called "reviewer.list...." 
Could they maybe be put together with those above submission.list.viewMore/less i.e. have something like common.list.viewMore/Less?
Some of the locale keys here also contain variables, that are not provided here, or?

Reply:
Good idea.